### ISSUE TITLE
Very hard to follow transaction history at an address level

### ISSUE THREAD

**Issue by AndySchroder** [goatpig/BitcoinArmory#121]
When going to wallet properties, there is a list of addresses and their current balance and how many transactions there were to that address. There is no way to know what the actual transactions were to/from that address. It would be nice to show the transactions to/from each address with a new pop up window when right clicking on each of those addresses and choosing "show transactions".

It would also be nice to show the receiving/sending address of the local wallet in a tooltip when hovering the mouse over a transaction in the main transaction history. It would also be nice to show the address label when clicking on "view details" for a transaction in the main transaction window.

**Comment by achow101** [comment]
> There is no way to know what the actual transactions were to/from that address.

You should be able to double click an address and see the transactions to and from that address as well as some other buttons for the address's keys.

> It would also be nice to show the receiving/sending address of the local wallet in a tooltip when hovering the mouse over a transaction in the main transaction history.

No. In most transactions, there are not just one "sending address" or just one "receiving address". Bitcoin is spent from different TXOs that map to different addresses.

**Comment by AndySchroder** [comment]
That's cool that you can actually get to a transaction history window by double clicking. I didn't realize the double click functionality was sensitive to the horizontal location of each row. Maybe it would be useful to make each of those double click functions also a right click menu item.

**Comment by AndySchroder** [comment]
Regarding TXOs: I know what you are saying, but the TXOs are associated with an address. When you go to the "Transaction Info" window for a transaction, armory currently lists the address, not the TXO in the input section.

I would think the tooltip could still show addresses part of the current wallet that are associated with the TXOs and UXTOs in the transaction.

I guess there could still be a problem with the tooltip showing the addresses though because the number of address could get very large, making the tooltip huge, in which case you'd have to only show the first 3 (or so) number of sending and receiving, then show a "..." to indicate the list is truncated.


Do you have a problem with my suggestion of "It would also be nice to show the address label when clicking on "view details" for a transaction in the main transaction window."?
